playground/infrastructure/datastore_client.py

```diff
@@ -78,37 +78,36 @@ def save_to_cloud_datastore(self, examples_from_rep: List[Example], sdk: Sdk):
         examples_ids_before_updating = self._get_all_examples(sdk)
 
         # loop through every example to save them to the Cloud Datastore
-        with self._datastore_client.transaction():
```

Comment:
I removed the transaction here because it caused the error related to datastore restriction during deleting an entity group.

Error:

`google.api_core.exceptions.InvalidArgument: 400 operating on too many entity groups in a single transaction.`

Doc:

https://cloud.google.com/datastore/docs/concepts/structuring_for_strong_consistency#entity_group_limitations_on_transactions